### ISSUE TITLE
Install error: libevent... configure: error: not found

### ISSUE THREAD

**Issue by wurenny** [pgbouncer/pgbouncer#88]
i have installed libevent2 by yum on my centos7, but when i run configure, it still said not found:
$ rpm -aql libevent
/usr/lib64/libevent-2.0.so.5
/usr/lib64/libevent-2.0.so.5.1.9
/usr/lib64/libevent_core-2.0.so.5
/usr/lib64/libevent_core-2.0.so.5.1.9
/usr/lib64/libevent_extra-2.0.so.5
/usr/lib64/libevent_extra-2.0.so.5.1.9
/usr/lib64/libevent_openssl-2.0.so.5
/usr/lib64/libevent_openssl-2.0.so.5.1.9
/usr/lib64/libevent_pthreads-2.0.so.5
/usr/lib64/libevent_pthreads-2.0.so.5.1.9
/usr/share/doc/libevent-2.0.21
/usr/share/doc/libevent-2.0.21/ChangeLog
/usr/share/doc/libevent-2.0.21/LICENSE
/usr/share/doc/libevent-2.0.21/README

$ ll /usr/lib64 |grep libevent
lrwxrwxrwx.  1 root root       21 Nov 11 06:41 libevent-2.0.so -> libevent-2.0.so.5.1.9
lrwxrwxrwx.  1 root root       21 Oct  7 12:14 libevent-2.0.so.5 -> libevent-2.0.so.5.1.9
-rwxr-xr-x.  1 root root   297792 Jun 13  2014 libevent-2.0.so.5.1.9
lrwxrwxrwx.  1 root root       26 Nov 11 06:42 libevent_core-2.0.so -> libevent_core-2.0.so.5.1.9
lrwxrwxrwx.  1 root root       26 Oct  7 12:14 libevent_core-2.0.so.5 -> libevent_core-2.0.so.5.1.9
-rwxr-xr-x.  1 root root   179800 Jun 13  2014 libevent_core-2.0.so.5.1.9
lrwxrwxrwx.  1 root root       27 Nov 11 06:43 libevent_extra-2.0.so -> libevent_extra-2.0.so.5.1.9
lrwxrwxrwx.  1 root root       27 Oct  7 12:14 libevent_extra-2.0.so.5 -> libevent_extra-2.0.so.5.1.9
-rwxr-xr-x.  1 root root   133864 Jun 13  2014 libevent_extra-2.0.so.5.1.9
lrwxrwxrwx.  1 root root       29 Nov 11 06:43 libevent_openssl-2.0.so -> libevent_openssl-2.0.so.5.1.9
lrwxrwxrwx.  1 root root       29 Oct  7 12:14 libevent_openssl-2.0.so.5 -> libevent_openssl-2.0.so.5.1.9
-rwxr-xr-x.  1 root root    24456 Jun 13  2014 libevent_openssl-2.0.so.5.1.9
lrwxrwxrwx.  1 root root       30 Nov 11 06:44 libevent_pthreads-2.0.so -> libevent_pthreads-2.0.so.5.1.9
lrwxrwxrwx.  1 root root       30 Oct  7 12:14 libevent_pthreads-2.0.so.5 -> libevent_pthreads-2.0.so.5.1.9
-rwxr-xr-x.  1 root root    11200 Jun 13  2014 libevent_pthreads-2.0.so.5.1.9

$ ./configure --prefix=/usr/local --with-libevent=/usr/lib64
..........
checking for libevent... configure: error: not found, cannot proceed

what should i do ?

**Comment by wurenny** [comment]
i found the libevent must be installed via source code manually...but not yum

**Comment by zakaryanla** [comment]
Через исходный код это как?

**Comment by GKurt** [comment]
I got this error after "yum install libevent", and it went away after I installed some other packages:

yum install libevent-devel
yum install libtevent-devel

I suspect that "libtevent-devel" is not necessary. I think the configure script really couldn't find some 'C' header files, not the libevent library.

Similarly, even though the openssl library was installed, the configure script error'ed out on it. That error went away after:

yum install openssl-devel openssl-perl openssl-static

and I was able to run "make" and "make install"
